### ISSUE TITLE
Pydantic v2 compatibility and conflicting dependencies

### ISSUE THREAD

**Issue by prrao87** [qdrant/qdrant-client#229]
I'm using Pydantic v2 upstream in some data pipelines, but the qdrant client itself seems to depend on Pydantic under the hood, and it's resulting in a dependency conflict in my virtual environment. Are there plans to release a version of `qdrant-client` with Pydantic v2 compatibility anytime soon? Thanks!

```
ERROR: Cannot install -r requirements.txt (line 1), pydantic~=2.0.0 and qdrant-client==1.3.1 because these package versions have conflicting dependencies.
The conflict is caused by:
    The user requested pydantic~=2.0.0
    qdrant-client 1.3.1 depends on pydantic<2.0 and >=1.8

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
    The user requested pydantic~=2.0.0
    qdrant-client 1.3.0 depends on pydantic<2.0 and >=1.8
```

**Comment by joein** [comment]
Hi @prrao87 
Yes, it is already merged and should be available soon (with the next release)

**Comment by joein** [comment]
@prrao87 try `qdrant-client==1.3.2`

**Comment by prrao87** [comment]
@joein It works! I also made a fix to the docs in #233 as the kwarg change from `top` to `limit` hasn't been updated there. Thanks for the quick response!
